examples/learn-and-lock/baseline_decoy_server.py

```
@@ -0,0 +1,96 @@
+        }
+
+    return {
+        "jsonrpc": "2.0",
+        "id": req_id,
+        "error": {"code": -32601, "message": f"Method not found: {method}"},
+    }
+
+
+def run_stdio() -> None:
+    for line in sys.stdin:
+        line = line.strip()
```

Ruff (0.15.21)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)